scripts/configure

```diff
@@ -102,6 +102,7 @@ ENV_FILE="./templates/.env"
 [[ ! -z ${PREV_ENV_FILE+x} ]] && [[ -f "${PREV_ENV_FILE}" ]] && source "${PREV_ENV_FILE}"
 
 # Umbrel
+BIND_ADDRESS="${UMBREL_BIND_ADDRESS:-BIND_ADDRESS:-0.0.0.0}"
```

Comment:
I think to reduce confusion and chance of bugs just stick to a single variable name instead of checking for `UMBREL_BIND_ADDRESS` externally and using `BIND_ADDRESS` internally.

Reply:
The thing is that the `BIND_ADDRESS` env var specified by the user is overwritten by the previous run value, as the env file is sourced just above. So once it is set for the first time, it can no longer be changed. 

This is the workaround I found. Can you think of any other?

Reply:
Ahh I see. I wouldn't worry about that in this PR. I think the `BITCOIN_NETWORK` and `NGINX_PORT` environment variables have the same issue.

But yeah, that behaviour is annoying and potentially dangerous. Although maybe we should fix that for everything in a follow up PR. We could just save them in a temp var directly before sourcing, and then overwrite them directly after sourcing.

Reply:
Make sense. I will make the changes 👌

Reply:
Resolved in 85ecd29.